gf-webapp/src/logic/rooms/roomPage/roomsBlock/roomsList/RoomItem.tsx

```diff
@@ -25,7 +27,8 @@ export const RoomItem = (props: RoomItemProps) => {
           alt="room"
         />
         <h3 className={styles.title3}>
```

Comment:
I created constants for titles that we are using in rooms and room pages. Because the styles for h1 and h2 are used twice in different scss modules, I created a mixin to avoid duplicating code.

Reply:
Good!